CONTRIBUTING.rst

```diff
@@ -1,37 +1,214 @@
 Contributing
 ============
 
-Running Tests
--------------
+Instructions for contributors
+-----------------------------
+
+In order to make a clone of the `GitHub <https://github.com/aio-libs/aiobotocore>`_ repo: open the link and press the
+"Fork" button on the upper-right menu of the web page.
+
+I hope everybody knows how to work with git and gitlab nowadays :)
+
+Workflow is pretty straightforward:
+
+  1. Clone the `GitHub <https://github.com/aio-libs/aiobotocore>`_ repo
+
+  2. Make a change
+
+  3. Make sure all tests passed
+
+  4. Make sure the coverage doesn't get worse.
+
+  5. Commit changes to own aiobotocore
+
+  6. Correct message for ``commit``. You need read (`Commit message format`_).
 
-.. _GitHub: https://github.com/aio-libs/aiobotocore
+  7. Correct name for ``branch``. You need read (`Branch name format`_).
 
-Thanks for your interest in contributing to ``aiobotocore``, there are multiple
-ways and places you can contribute.
+  8. Make pull request from github page for your branch against master branch
 
-Fist of all just clone repository::
+  9. Optionally make backport pull Request(s) for landing a bug fix
+     into released aiobotocore versions.
 
-    $ git clone git@github.com:aio-libs/aiobotocore.git
+  10. Add changes in file ``CHANGES.txt`` for this read (`Changelog update`_).
 
-Create virtualenv with at least python3.5 (older version are not supported).
-For example using *virtualenvwrapper* commands could look like::
+
+Preconditions for running aiobotocore test suite
+------------------------------------------------
+
+We expect you to use a python virtual environment to run our tests.
+
+There are several ways to make a virtual environment.
+
+If you like to use *virtualenv* please run:
+
+.. code-block:: shell
 
    $ cd aiobotocore
-   $ mkvirtualenv --python=`which python3.5` aiobotocore
+   $ virtualenv -p /usr/bin/python3 venv-aiobotocore
+   $ . venv-aiobotocore/bin/activate
+
+For standard python *venv-aiobotocore*:
+
+.. code-block:: shell
+
+   $ cd aiobotocore
```

Comment:
I think a better strategy going forward is using tox, so you can test against multiple python versions easily.  Thoughts?

Reply:
Start to "tox" is good for "CI", for development it is desirable to check on the new version. Here I agree with Andrey in his report https://www.youtube.com/watch?v=7KgihdKTWY4

Reply:
sorry, I don't understand what you're trying to say?  I don't understand russian either so that video doesn't help.  Is that the aiohttp author by chance, asvetlov?